frontend/src/components/About.tsx

```diff
@@ -21,6 +21,25 @@ const About = () => {
 export default About;
 
 const AboutWhatIs = () => {
+  const personas = [
+    {
+      name: 'Students',
+      imgSrc: '',
+    },
+    {
+      name: 'Creatives',
+      imgSrc: '',
+    },
+    {
+      name: 'Developers',
+      imgSrc: '',
```

Comment:
Add types for this array: 

Something like this:

type Persona = {
  name: string;
  imgSrc: string;
};